tools/mtouch/Application.cs

```diff
@@ -1186,6 +1186,20 @@ void DetectCodeSharing ()
 			Driver.Watch ("Detect Code Sharing", 1);
 		}
 
+		static bool CompareLists (List<string> a, List<string> b)
+		{
+			if (a == b)
+				return true;
+			if (a == null ^ b == null)
+				return false;
+			if (a.Count != b.Count)
+				return false;
+			for (var i = 0; i < a.Count; i++)
+				if (a [i] != b [i])
+					return false;
+			return true;
+		}
```

Comment:
Maybe using https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.sequenceequal?view=netframework-4.8 is a little nicer?